templates/workspace_services/databricks/.gitignore

```diff
@@ -0,0 +1 @@
+.cnab/
```

Comment:
gitignore inherits from the root one. you can remove this file.

Reply:
removed